DXRNonVanilla/DeusEx/Classes/DXRandoGameInfo.uc

```diff
@@ -127,3 +127,24 @@ event Super_PostLogin( playerpawn NewPlayer )
             }
     }
 }
+
+function SpawnDXRando(DeusExPlayer player)
+{
+    local DeusExLevelInfo DeusExLevelInfo;
+    local PlayerStart ps;
+
+    class'DeusExLevelInfo'.default.MapName = "ENDGAME4";
+    class'DeusExLevelInfo'.default.missionNumber = 99;
+    class'DeusExLevelInfo'.default.Script = class'MissionEndgame';
```

Comment:
This version of SpawnDXRando might as well just directly spawn the DXRMissionEndgame, rather than spawning the original and replacing it